invites/InvitesExample/AppDelegate.m

```diff
@@ -43,28 +43,25 @@ - (BOOL)application:(UIApplication *)application
             openURL:(NSURL *)url
   sourceApplication:(NSString *)sourceApplication
          annotation:(id)annotation {
-  // Handle App Invite requests
-  FIRReceivedInvite *invite =
-      [FIRInvites handleURL:url sourceApplication:sourceApplication annotation:annotation];
-  if (invite) {
-    NSString *matchType =
-        (invite.matchType == FIRReceivedInviteMatchTypeWeak) ? @"Weak" : @"Strong";
-    NSString *message =
-        [NSString stringWithFormat:@"Deep link from %@ \nInvite ID: %@\nApp URL: %@\nMatch Type:%@",
-                                   sourceApplication, invite.inviteId, invite.deepLink, matchType];
-
-    [[[UIAlertView alloc] initWithTitle:@"Deep-link Data"
-                                message:message
-                               delegate:nil
-                      cancelButtonTitle:@"OK"
-                      otherButtonTitles:nil] show];
-
+  if ([[GIDSignIn sharedInstance] handleURL:url
+                      sourceApplication:sourceApplication
+                                 annotation:annotation]) {
     return YES;
   }
-
-  return [[GIDSignIn sharedInstance] handleURL:url
-                             sourceApplication:sourceApplication
-                                    annotation:annotation];
+  // [START_EXCLUDE silent]
+  __weak AppDelegate *weakSelf = self;
```

Comment:
This probably doesn't need to be a weak variable, but it's inconsequential to the snippets.

Reply:
Normally I would prefer to have weak var here. But taking into account that this is quickstart sample, so that developers can quickly start using the library, I agree with Morgan and prefer not to complicate this code without need.